executor/show.go

```diff
@@ -1062,6 +1062,20 @@ func ConstructResultOfShowCreateTable(ctx sessionctx.Context, tableInfo *model.T
 		buf.WriteString("*/")
 	}
 
+	if tableInfo.StatsOptions != nil {
+		cols := make([]string, 0, len(tableInfo.StatsOptions.ColumnList))
+		for _, col := range tableInfo.StatsOptions.ColumnList {
+			cols = append(cols, col.L)
+		}
+		fmt.Fprintf(buf,
+			" /*T! STATS_BUCKETS=%d,STATS_TOPN=%d,STATS_SAMPLE_RATE=%s,STATS_COL_CHOICE='%s',STATS_COL_LIST='%s' */",
```

Comment:
Would it be better if we show `STATS_OPTION_NAME=DEFAULT` for the default case or just erase them?

Reply:
Updated.